aunt-leahs-app/src/components/locationTable.jsx

```diff
@@ -27,41 +28,39 @@ class LocationSelection extends React.Component {
         };
 
         this.selectLocation = this.selectLocation.bind(this);
-
     }
 
-    selectLocation(id){
+    selectLocation(id) {
         var updatedLocations = this.state.locations.slice(0);
         updatedLocations.forEach(l => {
```

Comment:
Use descriptive naming for variables. A better name than l might be location.

Reply:
Resolved in latest commit